docs/content/references/config/database.md

```diff
@@ -7,12 +7,48 @@ We officially support and test these databases:
-|-----------------------|---------|---------------------|--------------------------------------------------------------------------------------------|
-| `HD_DATABASE_TYPE`    | -       | `postgres`          | The database type you want to use. This can be `postgres`, `mysql`, `mariadb` or `sqlite`. |
-| `HD_DATABASE_NAME`    | -       | `hedgedoc`          | The name of the database to use. When using SQLite, this is the path to the database file. |
-| `HD_DATABASE_HOST`    | -       | `db.example.com`    | The host, where the database runs. *Only if you're **not** using `sqlite`.*                |
-| `HD_DATABASE_PORT`    | -       | `5432`              | The port, where the database runs. *Only if you're **not** using `sqlite`.*                |
-| `HD_DATABASE_USER`    | -       | `hedgedoc`          | The user that logs in the database. *Only if you're **not** using `sqlite`.*               |
-| `HD_DATABASE_PASS`    | -       | `password`          | The password to log into the database. *Only if you're **not** using `sqlite`.*            |
+
+| environment variable | default | example          | description                                                                                |
+| -------------------- | ------- | ---------------- | ------------------------------------------------------------------------------------------ |
+| `HD_DATABASE_TYPE`   | -       | `postgres`       | The database type you want to use. This can be `postgres`, `mysql`, `mariadb` or `sqlite`. |
+| `HD_DATABASE_NAME`   | -       | `hedgedoc`       | The name of the database to use. When using SQLite, this is the path to the database file. |
+| `HD_DATABASE_HOST`   | -       | `db.example.com` | The host, where the database runs. _Only if you're **not** using `sqlite`._                |
+| `HD_DATABASE_PORT`   | -       | `5432`           | The port, where the database runs. _Only if you're **not** using `sqlite`._                |
+| `HD_DATABASE_USER`   | -       | `hedgedoc`       | The user that logs in the database. _Only if you're **not** using `sqlite`._               |
+| `HD_DATABASE_PASS`   | -       | `password`       | The password to log into the database. _Only if you're **not** using `sqlite`._            |
+
+<!-- markdownlint-enable proper-names -->
+
+### SSL/TLS configuration for postgres database
+
+<!-- markdownlint-disable proper-names -->
+
+| environment variable                  | default | type   | description                                                                                                   |
+| ------------------------------------- | ------- | ------ | ------------------------------------------------------------------------------------------------------------- |
+| `HD_DATABASE_SSL_CA`                  | -       | string | The SSL/TLS certificate key needed for database connection.                                                   |
```

Comment:
Maybe specify more concrete: Do you pass the file name of the CA certificate or do you pass the contents (and if so in which encoding)?

Please state that this certificate (as every CA cert) is the anchor for checking the validity of the cert chain.